guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage.adoc

```diff
@@ -0,0 +1,36 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-with-external-storage"]
+= Preparing for disaster recovery with active and passive {ProjectServer} with external storage
+
+Create your passive {ProjectServer} as a clone of your active {ProjectServer}.
+Make sure that both servers connect to the database content on your network-attached storage (NAS).
+
+.Procedure
+. Replicate the `/var/lib/pulp` and `{postgresql-lib-dir}` directories from the active {ProjectServer} to your NAS.
+// Do we want to keep this generic like the virtualization scenario, or is this a good place to provide example rsync commands or something?
+. Clone your active {ProjectServer}.
+See xref:cloning_satellite_server[].
++
+Keep the source server powered on and power off the new cloned server.
```

Comment:
`and` might imply that instruct the reader to do these in a particular sequence. I don't think you do, so perhaps this would be better:

`Keep the source server powered on. Keep the new cloned server powered off.`